onnxruntime/test/providers/cpu/math/matmul_integer_test.cc

```diff
@@ -311,9 +312,15 @@ void RunMatMulIntegerU8S8Test(const int M, const int N, const int K, bool non_ze
   }
 }
 
-#define RUN_MATMUL_INTEGER_U8S8(M, N, K)                    \
-  RunMatMulIntegerU8S8Test(M, N, K, false /*non_zero_zp*/); \
-  RunMatMulIntegerU8S8Test(M, N, K, true /*non_zero_zp*/);
+#define RUN_MATMUL_INTEGER_U8S8(M, N, K)                                                         \
```

Comment:
nit: should the macro and test names be updated to reflect that this is now U8X8 and Uint8_Xint8?